core/server/common/src/main/java/alluxio/master/audit/AuditContext.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.master.audit;
+
+import java.io.Closeable;
+
+/**
+ * Context for Alluxio audit logging.
+ */
+public interface AuditContext extends Closeable {
+
+  /**
+   * Set to true if the operation associated with this {@link AuditContext} is allowed, false
+   * otherwise.
+   *
+   * @param allowed true if operation is allowed, false otherwise
+   * @return {@link AuditContext} instance itself
+   */
+  AuditContext setAllowed(boolean allowed);
+
+  /**
+   * Set to true if the operration associated with this {@link AuditContext} is allowed and
+   * succeeds.
+   *
+   * @param success true if the operation is successful, false otherwise
+   * @return {@link AuditContext} instance itself
+   */
+  AuditContext setStatus(boolean success);
```

Comment:
Rename to `setSuccess` - it isn't clear what "true" means as a status

optionally rename to `reportSuccess` and remove the argument since all callers pass `true` anyway